tests/tests_integration/tests_db_connection_integration/test_clickhouse_integration.py

```diff
@@ -321,3 +321,38 @@ def func_finalizer():
     # wrong syntax
     with pytest.raises(Exception):
         clickhouse.execute(f"CREATE FUNCTION wrong_function AS (a, b) -> {suffix}")
+
+
+def test_clickhouse_connection_no_jdbc_dialect(spark, processing, load_table_data, caplog):
+    clickhouse = Clickhouse(
+        host=processing.host,
+        port=processing.port,
+        user=processing.user,
+        password=processing.password,
+        database=processing.database,
+        spark=spark,
+    )
+
+    table = load_table_data.full_name
+    clickhouse.get_df_schema(table)
+
+    with caplog.at_level(logging.INFO):
+        assert "Detected dialect: 'org.apache.spark.sql.jdbc.NoopDialect'" in caplog.text
```

Comment:
Maybe those asserts could be moved to few existing tests?

Reply:
moved asserts except `.get_df_schema` as there are not suitable tests for this method in postgres/clickhouse tests

Reply:
This could be moved somewhere to test_clickhouse_reader test

Reply:
however `get_df_schema` is not executed in any `run` method explicitly except `Hive.write_df_to_target` where `.get_df_schema` is overriden, therefore there are no logs niether for reading nor for writing operations

Reply:
Hm, `.get_df_schema` is called only if DBReader(hwm=...) is used

Reply:
moved assertion to `test_postgres_strategy_incremental_batch_different_hwm_type_in_store`